### ISSUE TITLE
Context menu reveal in file tree removed

### ISSUE THREAD

**Issue by moogus** [facebook/nuclide#1073]
### Issue and Steps to Reproduce

OMG why do you keep breaking things!!! Everytime I update nuclide something breaks or disappears.....**😞**

- open a tab 
-  right click that tab
- context menu item "reveal in file tree" longer there

![image](https://cloud.githubusercontent.com/assets/4889659/24457230/7ffc102a-148d-11e7-94a4-5da9bb06af67.png)

### Expected Behavior
It should have the context menu item "Reveal in file tree"

### Actual Behavior
It doesn't

### Versions
* Atom: 1.15.0
* Nuclide: 0.212.0
* Client OS:Mac Sierra
* Server OS (optional):

**Comment by tjfryan** [comment]
I'm not able to reproduce this bug.  Are there any other steps I might need to do?
Also, does the path of the file appear in a tooltip when you hover over the file tab?

**Comment by moogus** [comment]
Yes the path does appear but the context menu item has been removed.  When attempting to reproduce it do you see a context item 'reveal in file tree', if you don't then you have reproduced it!
